units/src/iosMain/kotlin/dev/icerock/moko/units/UnitCollectionViewDataSource.kt

```diff
@@ -14,17 +14,20 @@ import platform.darwin.NSInteger
 import platform.darwin.NSObject
 
 @ExportObjCClass
-class UnitCollectionViewDataSource(
-    private val collectionView: UICollectionView
```

Comment:
i suggest backward compatible change:
```kotlin
class UnitCollectionViewDataSource(
    private val collectionView: UICollectionView,
    private val reloadDataHandler: (UICollectionView, oldData: List<CollectionUnitItem>?, newData: List<CollectionUnitItem>?) -> Unit = { collectionView, _, _ -> collectionView.reloadData() }
)
```
and in setter of `unitItems`:
```kotlin
val old = field
field = value
if (value != null) unitsRegistry.registerIfNeeded(value)
reloadDataHandler(collectionView, old, value)
```

Reply:
and same to TableViewDataSource

Reply:
it's allow configure any reload handler from outside and have default implementation, same as in old version